File: card-classes/log/render.py
import mica
import markdown

@mica.render
def render(content, config):
    """Render the activity log card."""
    html = markdown.markdown(content, extensions=["tables", "fenced_code", "sane_lists"])
    return f'''
    <div class="card-log">
        <div class="card-markdown">{html}</div>
    </div>
    '''

File: card-classes/log/render.js
/**
 * Log card class — renders the activity log.
 */

import { marked } from 'marked';

export default function render(content, config) {
  const html = marked.parse(content, { breaks: false, gfm: true });
  return `
    <div class="card-log">
      <div class="card-markdown">${html}</div>
    </div>
    <script>
      const unsub = mica.on('file-changed', (e) => {
        if (e.filename === mica.filename) mica.refresh();
      });
      mica.onDestroy(() => unsub());
    </script>
  `;
}
